Dockerfile.template

```diff
@@ -74,7 +74,8 @@ RUN echo y | ${CMDLINE_TOOLS_ROOT}/sdkmanager "platforms;android-33" && \
 # Install some useful packages
 RUN echo y | ${CMDLINE_TOOLS_ROOT}/sdkmanager "extras;android;m2repository" && \
 	echo y | ${CMDLINE_TOOLS_ROOT}/sdkmanager "extras;google;m2repository" && \
-	echo y | ${CMDLINE_TOOLS_ROOT}/sdkmanager "extras;google;google_play_services"
+	echo y | ${CMDLINE_TOOLS_ROOT}/sdkmanager "extras;google;google_play_services" && \
+	gem install fastlane
```

Comment:
Do you know anything about Fastlane and how frequently it updates and breaking changes?

Typically for CircleCI images, libraries and software that ships with the OS we install as is, but 3rd party software we like to pin down to a specific version. This helps make builds deterministic.